packages/atti-components/src/components/Slider/styles/SliderStep.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import SmallText from "../../Typographies/SmallText";
+import PropTypes from "prop-types";
+
+const SliderStep = styled.span`
+  ${({theme: {components: {slider: {sliderStep}}}}) => `
+    background-color: ${sliderStep.colors.backgroundColor};
+    height: ${sliderStep.sizes.height};
+    width: ${sliderStep.sizes.width};
+    
+    ${SmallText} {
+      display: flex;
+      justify-content: center;
+      margin-top: ${sliderStep.title.margins.marginTop};
+    }
+  `}
+  left: ${({leftPosition}) => leftPosition + '%'};
+  position: absolute;
+  z-index: 1000;
```

Comment:
We should create an `elevations` property in the theme and use it as a reference for our z-index

Reply:
Done